cli/commands/genesis/root.go

```diff
@@ -79,6 +79,15 @@ func GetGenesisValidatorRootCmd(cs chain.Spec) *cobra.Command {
 					math.Gwei(cs.EffectiveBalanceIncrement()),
 					math.Gwei(cs.MaxEffectiveBalance()),
 				)
+
+				// mimic processGenesisActivation
+				minEffectiveBalance := math.Gwei(
```

Comment:
this is technically some more code duplication, the same that caused the CLI to break.
I will look for a way to consolidate the code more, so that we reuse code from state_processor

Reply:
Also, if we cannot remove code duplication we should have UTs testing this, Ideally with quick. Will work on this